john-co/liferay-portal
# LPS-X Use translation manager in journal-web

**Test plan**
- Fetch this branch
- Deploy the `journal-web` OSGi module
- Navigate to Content > Web Content
- Create a new Web Content

You should see a "manage translation" button

![](https://user-images.githubusercontent.com/5572/132643732-6f80b156-6f76-4915-8ad9-4d95f69b28c3.png)

When clicked, a modal should appear 

![](https://user-images.githubusercontent.com/5572/132643817-b5c23355-d343-44c1-954c-6e01c43efc3e.png)
